### ISSUE TITLE
New version

### ISSUE THREAD

**Issue by Zero3K** [wooxo/Vhd2Disk#8]
It would be nice if you made one soon that includes my suggestions.

**Comment by orkblutt** [comment]
Sorry but I don't have lot of free time.
Are you using this tool in a commercial way ?

**Comment by Zero3K** [comment]
No, I am just wanting to use something else besides Drive Snapshot that uses a non-propitiatory disk image format. You could include a Sponsor page/donate button on this repository if you're wanting money.

**Comment by orkblutt** [comment]
No. Money is not the point here. TIme is...

**Comment by Zero3K** [comment]
Oh, okay.

**Comment by orkblutt** [comment]
I'll do my best in the next few days... but be patient ;)

**Comment by Zero3K** [comment]
Thanks.

**Comment by Zero3K** [comment]
Any news?

**Comment by micwoj92** [comment]
Any news?

**Comment by Zero3K** [comment]
It would be nice if someone could work on it soon.

**Comment by Zero3K** [comment]
Bump
